src/components/swap/SwapModalDetailRow.tsx

```diff
@@ -0,0 +1,38 @@
+import Row from 'components/Row'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { ReactNode } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { Label } from './SwapModalHeaderAmount'
+
+const Value = styled.span<{ color?: string }>`
+  color: ${({ color, theme }) => color ?? theme.textPrimary};
+  text-align: end;
+  max-width: 45%;
```

Comment:
if this component has a max-width then it should be a `div` and not a `span` - spans are for non-layout wrappers